.github/workflows/build.yml

```diff
@@ -55,4 +55,4 @@ jobs:
     - name: Install Opctl
       run: curl -L https://github.com/opctl/opctl/releases/latest/download/opctl-linux-amd64.tgz | sudo tar -xzv -C /usr/local/bin
 
-    - run: opctl run -a gitBranch=${{ steps.branch_name_push.outputs.branch }}${{ steps.branch_name_pr.outputs.branch }} build
+    - run: opctl run -a gitBranch=${{ steps.branch_name_push.outputs.branch }}${{ steps.branch_name_pr.outputs.branch }} -a version=0.0.1 build
```

Comment:
what's the purpose of adding `-a version=0.0.1` to the build command via this build action? It confuses me since the `build op` had the `version` property removed.

Reply:
The `test` op had `version` remove, not the build op. The build op had the default value for `version` removed. The only place that the `version` actually matters is during the release op - but it has to be defined in CI even though the release op isn't run in CI

Reply:
I saw it removed from the `test/inputs` section and missed it still present in the op's `inputs` section. Collapsing PR reviews 😓